packages/tab/src/Tab.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+
+const Tab = ({ block, className, children, ...props }) => {
+  const classNames = classnames('tab', className, {
+    'tab-block': block
+  })
+  return (
+    <ul className={classNames} {...props}>
+      {children}
+    </ul>
+  )
+}
+Tab.propTypes = {
+  block: PropTypes.bool,
+  className: PropTypes.string,
+  children: PropTypes.any
+}
+
+const Item = ({ active, action, className, children, ...props }) => {
```

Comment:
@ccontreras I prefer to have each component in separate files. I will try to merge it and do the change because it has been 1 week since you did the contribution. :)

Reply:
Thank you @rmariuzzo, i'll keep it in mind next time.

Reply:
Hey @rmariuzzo, would you mind if I do this for you?